packages/light-client/package.json

```diff
@@ -77,6 +77,8 @@
   },
   "devDependencies": {
     "@chainsafe/as-sha256": "^0.3.1",
+    "@types/qs": "^6.9.7",
```

Comment:
added as dev deps as we use it in tests `packages/light-client/test/utils/server.ts`